torch/csrc/jit/script/function_schema_parser.cpp

```diff
@@ -158,6 +158,8 @@ struct SchemaParser {
           return static_cast<int64_t>(at::kStrided);
         } else if ("Mean" == text) {
           return static_cast<int64_t>(Reduction::Mean);
+        } else if ("contiguous_format" == text) {
+          return static_cast<int64_t>(c10::MemoryFormat::Contiguous);
```

Comment:
Shouldn't the other format types be supported in this switch?

Reply:
Same as 
```
 } else if ("Mean" == text) {
          return static_cast<int64_t>(Reduction::Mean);
```
we can introduce all options right now, or add them when we need them in native_functions.yaml